### PR TITLE
Update table-row-context-example.html

### PR DESCRIPTION
**Pull request by fileto1**
Fixed comments for each column.

### CODE REVIEW

**Comment by fileto1** [comment]
@googlebot I signed it!

File: src/components-examples/material/table/table-row-context/table-row-context-example.html [hunk 1]
```diff
@@ -1,5 +1,5 @@
 <table mat-table [dataSource]="data" class="mat-elevation-z8">
-  <!-- Index Column -->
+  <!-- Implicit Column -->
```
Comment (jelbourn) [inline src/components-examples/material/table/table-row-context/table-row-context-example.html]: This one should stay as "Index"

Comment (fileto1) [reply src/components-examples/material/table/table-row-context/table-row-context-example.html]: But already exists the index column, do you can see in the images. Implicity is another column.
image
image

Comment (jelbourn) [reply src/components-examples/material/table/table-row-context/table-row-context-example.html]: Ah, I thought that `$implicit` was index here, but I was mistaken

**Pull request merged**